main.go

```diff
@@ -178,7 +192,18 @@ func main() {
 	}
 
 	setupLog.Info("starting manager")
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
+	ctx := ctrl.SetupSignalHandler()
+	errChan := make(chan error, 1)
+	go func(chan error) {
+		if err := mgr.Start(ctx); err != nil {
+			errChan <- err
+		}
+	}(errChan)
+
+	setupLog.Info("starting backfill")
```

Comment:
I think we can improve this message a bit.

Reply:
updated to `restoring flagd-kubernetes-sync cluster role binding subjects from current cluster state`